source/crud/tests/v2/aggregate-out-readConcern.yml

```diff
@@ -0,0 +1,155 @@
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+
+minServerVersion: '4.1'
+
+collection_name: &collection_name 'testaggcollection'
+
+tests:
+  -
+    description: "readConcern majority with out stage"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern:
+            level: "majority"
+        arguments:
+          pipeline:
+            - $sort: {x : 1}
+            - $match:
+                _id: {$gt: 1}
```

Comment:
Small suggestion to inline these as you did in the APM expectations below (`{ _id: { $gt: 1 } }`).